lib/_http_server.js

```diff
@@ -490,6 +490,10 @@ Server.prototype.closeIdleConnections = function() {
   const connections = this[kConnections].idle();
```

Comment:
I'm confused about the definition of `this[kConnections].idle()`. If it has an incoming message in the socket, why it will also defined as idleConnections?

Reply:
I know, it's a bit confusing.
The definition of "idle", "active", "expired" and so forth must be interpreted from the point of view of the server inbound parser.

In this case a socket is considered idle because it has finished sending the request which means that (ignoring HTTP pipelining)no new data is expected to be received. The socket is therefore either waiting for a response or in HTTP Keep-Alive mode.

Is that clear now?